vassal-app/src/main/java/VASSAL/script/ExpressionInterpreter.java

```diff
@@ -392,6 +392,12 @@ public Object getLocalizedProperty(String name) {
     return value == null ? "" : wrap(value.toString());
   }
 
+  /** getProperty minus the wrap */
+  public Object getString(String name) {
+    final Object value = source.getProperty(name);
+    return value == null ? "" : value.toString();
+  }
```

Comment:
Something I discovered just now is that you could also do this with `Objects.toString()`:

```
return Objects.toString(source.getProperty(name), "");
```

Reply:
Want me to change it to that? I'd just simplified the existing getProperty() code to not call "wrap"

Reply:
Not saying you have to, just pointing out that it's a thing. If I were starting fresh, it's a thing I expect I'd use a lot.